src/quo2/components/account_selector.cljs

```diff
@@ -0,0 +1,80 @@
+(ns quo2.components.account-selector
+  (:require
+   [quo.theme :as theme]
+   [quo.react-native :as rn]
+   [quo2.foundations.colors :as colors]))
+
+(def themes
+  {:light {:default         {:bg      colors/neutral-10
+                             :account-text     colors/black
+                             :label-text     colors/neutral-50}
+           :transparent     {:bg      colors/neutral-80-opa-5
+                             :account-text     colors/black
+                             :label-text     colors/neutral-50}}
+
+   :dark  {:default         {:bg      colors/neutral-80-opa-80
+                             :account-text    colors/white
+                             :label-text     colors/neutral-40}
+           :transparent     {:bg      colors/white-opa-5
+                             :account-text    colors/white
+                             :label-text     colors/neutral-40}}})
+
+;we prefer using temp uri for assets so that un-neccessary assets do not become a part of release bundle
+(def temp-fire-icon-uri "https://s3-alpha-sig.figma.com/img/58f0/9735/781a2f7ece5e5c0fde2cb7e9d2c22c77?Expires=1661731200&Signature=f5hvtb9Wz1Zwd4-dft9gOyBP25rR1wrVFtHyCQh9NCslCBZ7JJzex7qYdYZn2Qvf6ep50HIdkO1oh6wVBqeds1BrX0T4ojSzM8HA0CQkXlVveEwEyjcqb~XPx3m2zMFEcaoUX87Ho3I6ZHK477WwoHgKX9JwfpfoNZBSqVT1TiTiTPpNdv4AEY-5TJcuZk7mS1-O1eCuEoQA-VTWX3UXExsZ9wip-rnrp2o8a7dx1run9jHSNqsKfEMgK2Mc3zjphzW34asgrxCf1plRv3ZD12ht~pzsuph8FeeBb9HFLtumbUKbDI7L6sffEFGY6msxUBr0ret0akFlk1x00gnwCA__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA")
```

Comment:
thats interesting

Reply:
how did you share it ?

Reply:
Got it from figma, I'll show it on today's call

Reply:
In figma If you click on the highlighted file it would open the asset in the browser which would then contain the url for the asset/icon/image

![Screenshot 2022-08-19 at 2 39 47 PM](https://user-images.githubusercontent.com/64726664/185586154-dc89ab4e-996f-479f-b5ae-28f900f8f327.png)